aten/src/ATen/native/mps/operations/Eye.mm

```diff
@@ -54,7 +54,9 @@
   // Derive from MPSCachedGraph
   // This structure is used to cache an MPSGraph with certain keys, so that we don't have to compile the same MPSGraph
   // time and time again for the same operation The keys of this structure are based on the inputs and outputs needed
-  // for the operation Here, we don't have any input tensors, just an output tensor
+  // for the operation Here, we don't have any input tensors, just an output tensor If the operator to be added is unary
+  // or binary, instead of creating a new CachedGraph struct yourself, please consider using `MPSUnaryCachedGraph` or
+  // `MPSBinaryCachedGraph` and their corresponding Grad versions in `OperationUtils.h`.
```

Comment:
```suggestion
  // for the operation here, we don't have any input tensors, just an output tensor.
  // If the operator to be added is unary or binary, instead of creating a new CachedGraph struct yourself, please
  // consider using `MPSUnaryCachedGraph` or `MPSBinaryCachedGraph` and their corresponding Grad versions in
  // `OperationUtils.h`.
```